src/layout.rs

```diff
@@ -27,12 +27,67 @@ pub enum Direction {
     Vertical,
 }
 
+/// Constraints to apply  
+/// Currently there is no error or panic when a constraint cannot be fulfilled, and behavior for that is undefined (see <https://github.com/ratatui-org/ratatui/issues/354>)
```

Comment:
> See [#354 (comment)](https://github.com/ratatui-org/ratatui/issues/354#issuecomment-1666512893) for some detail on the panic. I was mistaken.

what should i change it to instead?

Reply:
It's the other functions that use the constraint that panic not the constraint itself (right?), so perhaps just omit the note about panics here.